### PR TITLE
Add `prometheus` metrics related to the `storage`.

### PR DESCRIPTION
**Pull request by MathieuDutSik**
## Motivation

The introduction of storage fees leads to the introduction of corresponding metrics for the storage that can be tracked via Prometheus.

## Proposal

The implementation is very straightforward.

## Test Plan

The CI should take care of it.

## Release Plan

- Need to bump the major/minor version number in the next release of the crates.
- Need to update the developer manual.
- This PR is adding or removing Cargo features.
- Release is blocked and/or tracked by other issues (see links below)

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

File: linera-chain/src/chain.rs [hunk 1]
```diff
@@ -71,6 +71,36 @@ pub static WASM_FUEL_USED_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
     .expect("Counter can be created")
 });
 
+pub static WASM_NUM_READS_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "wasm_num_reads_per_block",
+        "Wasm num reads per block",
```
Comment (ma2bd) [inline linera-chain/src/chain.rs:77]: Isn't this line supposed to be a description in English?

File: linera-chain/src/chain.rs [hunk 2]
```diff
@@ -71,6 +71,36 @@ pub static WASM_FUEL_USED_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
...
+        // Can add labels here
```
Comment (ma2bd) [inline linera-chain/src/chain.rs:78]: Remove comment?

File: linera-chain/src/chain.rs [hunk 3]
```diff
@@ -71,6 +71,36 @@ pub static WASM_FUEL_USED_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
...
+        &[]
+    )
+    .expect("Counter can be created")
+});
+
+pub static WASM_BYTES_READ_PER_BLOCK: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "wasm_bytes_read_per_block",
+        "Wasm bytes read per block",
+        // Can add labels here
```
Comment (ma2bd) [inline linera-chain/src/chain.rs:88]: same comments here and below

File: linera-chain/src/chain.rs (hunk 2 above)
Comment (MathieuDutSik) [reply linera-chain/src/chain.rs:78]: Ok.

File: linera-chain/src/chain.rs (hunk 3 above)
Comment (MathieuDutSik) [reply linera-chain/src/chain.rs:88]: Corrected.

File: linera-chain/src/chain.rs (hunk 1 above)
Comment (MathieuDutSik) [reply linera-chain/src/chain.rs:77]: Yes, done.

**Review by ma2bd — APPROVED** [review]

**Pull request merged**